morango/sync/syncsession.py

```diff
@@ -690,17 +698,6 @@ def __init__(self, *args, **kwargs):
         self.local_context.update(is_push=True)
         self.remote_context.update(is_push=True)
 
-    def initialize(self, sync_filter):
-        """
-        :param sync_filter: Filter
-        """
-        super(PushClient, self).initialize(sync_filter)
-
-        self.sync_connection._update_transfer_session(
```

Comment:
Where does the records_total get set now?

Reply:
It's in the [`NetworkQueueOperation`](https://github.com/learningequality/morango/pull/128/files#diff-dce2c6c78fa132287ff3004fb1a3eed42377dc787b8bf4986d544d501be22c9dR1121) which should occur after queuing locally